MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,12 +20,18 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onLoadFeedErrorStateChange: Observer<String?>?
```

Comment:
Since there's no other `on...ErrorStateChange` the name can be less specific. It will make the code shorter and also it will hide the implementation detail from the view model clients.

```suggestion
	var onErrorStateChange: Observer<String?>?
```

Reply:
done